sprout/sprout.sh

```diff
@@ -50,6 +52,17 @@ function needs_update() {
     return $RESULT
 }
 
+function update_yamls() {
+    cddir
```

Comment:
This is a job for another PR, but instead of calling cddir in every function, it's usually enough to cd `dirname $0` at the head of the file and use the dir stack (pushd/popd) to manage moving around.